sabelotodo/routes.py

```diff
@@ -14,5 +13,14 @@ def all_items():
     return jsonify([asdict(i) for i in Item.query.all()])
 
 
+@app.route('/item/<itemid>')
+def item_by_id(itemid):
+    if itemid.isnumeric():
+        rv = Item.query.get(itemid)
```

Comment:
what does rv stand for? Retrieved value? I'd make this a clearer variable name, but it's a total nitpick.